__tests__/app.test.js

```diff
@@ -56,6 +56,9 @@ describe("Test /api/articles/:article_id", () => {
         expect(body.topic).toBe("mitch");
         expect(body.author).toBe("butter_bridge");
         expect(body.votes).toBe(100);
+        expect(body).toMatchObject({
+          comment_count: expect.any(String),
```

Comment:
this needs to be in a separate test block, as a rule of thumb we don't change older tests